packages/deck-scripts/config/basePluginRollupConfig.js

```diff
@@ -0,0 +1,55 @@
+const baseRollupConfig = require('./baseRollupConfig');
+const externalGlobals = require('rollup-plugin-external-globals');
+
+module.exports = {
+  ...baseRollupConfig,
+  input: 'src/index.ts',
+  output: [{ dir: 'build/dist', format: 'es', sourcemap: true }],
+  plugins: [
+    ...baseRollupConfig.plugins,
+    // map imports from shared libraries (react, etc) to global variables exposed by spinnaker
+    externalGlobals(spinnakerSharedLibraries()),
```

Comment:
I'm assuming that we currently use globals. Do you know why by any chance?

Reply:
The globals are made available to the plugins. @christopherthielen might know this better, but I believe this is needed since plugins are built and loaded separately from `deck` and these globals are the way to make them available to them. If not, plugin rollup builds will have `import React from 'react'` statements and they won't work.